templates/_helpers.tpl

```diff
@@ -219,6 +219,18 @@ Determine name for cert auth secret
 {{ include "timothy.fullname" . }}-master-cert
 {{- end -}}
 
+{{- define "timothy.vectorLogFolder" -}}
+/etc/shared/logs
+{{- end -}}
+
+{{- define "timothy.vectorConfigPath" -}}
+/etc/shared/vector
+{{- end -}}
+
+{{- define "timothy.vectorConfigFile" -}}
+{{ include "timothy.vectorConfigPath" . }}/vector.toml
+{{- end -}}
+
```

Comment:
Could probably just do these as variables in the deployment file. Not an issue either way though